src/main/java/bio/terra/cli/command/cromwell/GenerateConfig.java

```diff
@@ -0,0 +1,36 @@
+package bio.terra.cli.command.cromwell;
+
+import bio.terra.cli.businessobject.Context;
+import bio.terra.cli.command.shared.BaseCommand;
+import bio.terra.cli.command.shared.options.CromwellPath;
+import java.util.List;
+import java.util.Optional;
+import picocli.CommandLine;
+import picocli.CommandLine.Command;
+
+/** This class corresponds to the third-level "terra cromwell generate-config" command. */
+@Command(
+    name = "generate-config",
+    description = "Autogenerate a cromwell.conf under the user specified path.")
+public class GenerateConfig extends BaseCommand {
+
+  @CommandLine.Mixin CromwellPath cromwellPath;
+
+  @Override
+  protected void execute() {
+    String googleProjectId = Context.requireWorkspace().getGoogleProjectId();
+    String petSaEmail = Context.requireUser().getPetSaEmail();
+
+    // Use {WORKSPACE_BUCKET} as a space holder now, until the workspace created with bucket.
+    Context.getConfig()
+        .getCommandRunnerOption()
+        .getRunner()
+        .runToolCommand(
```

Comment:
Follow the `terra app execute` [code](https://github.com/DataBiosphere/terra-cli/blob/main/src/main/java/bio/terra/cli/command/app/Execute.java#L30)the `runToolCommand` does not have return, so it could not deal with the error(like the path is not a real path). Also the `cromwellPath.path` is a string type, not sure if there any better way to require the "real path" input.

Reply:
i think that's oK. it will still print out error right so the user can re-run the command with a real path.